pkgs/development/python-modules/bidict/default.nix

```diff
@@ -36,12 +36,11 @@ buildPythonPackage rec {
     typing-extensions
   ];
 
-  pytestFlagsArray = [
-    # Pass -c /dev/null so that pytest does not use the bundled pytest.ini, which adds
-    # options to run additional integration tests that are overkill for our purposes.
-    "-c"
-    "/dev/null"
-  ];
+  # Overwrite the bundled pytest.ini, which adds options to run additional integration
+  # tests that are overkill for our purposes.
+  preBuild = ''
+    echo > pytest.ini
```

Comment:
```suggestion
  preCheck = ''
    rm pytest.ini
```

Reply:
Thanks @SuperSandro2000, done in the latest revision.